### PR TITLE
French version added and minor correction done on german and English …

### PR DESCRIPTION
**Pull request by grolimur**
French translation added   
minor corrections done on German and English versions

### CODE REVIEW

**Comment by stweil** [comment]
Merci bien.

File: README.md [hunk 1]
```diff
@@ -57,13 +57,13 @@ Zotkat ist eine Erweiterung von Zotero für die Katalogisierung und enthält auc
...
- * [`PicaGBV.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/PicaGBV.js) : Hierbei handelt es sich um einen Export-Translator mit dem Ziel bibliographische Daten aus Zotero ins Picaformat des GBV zu transformieren. Angepasst ist er für den Anwendungsfall: Aufsatzkatalogisierung von Buchkapiteln aus E-Books.
- * [`Wikidata QuickStatements.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/Wikidata%20QuickStatements.js) : Hierbei handelt es sich  um einen Export-Translator welcher Einträge aus Zotero in das Eingabeformat des Quickstatements-Tool transformiert.
- * [`Zotero JSON.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/Zotero%20JSON.js) : Dieser Export Translator ist vorallem zum Testen und liefert serialisierte JSON Objekte von Zotero aus.
+Momentan bietet zotkat folgende zusätzliche Zotero Translator an:
```
Comment (stweil) [inline README.md]: Are these additional blanks at the end of the line needed? See also other similar changes later.

**Comment by grolimur** [comment]
Not especially. I needed them to have it displayed properly in Markdown editor and I forgot to remove them.

**Comment by zuphilip** [comment]
Great! Thank you very much @grolimur! I can take it from here and fix the spacing. Maybe a little later because I am distracted with the conference and other things...

**Comment by zuphilip** [comment]
Merged :tada:  ! Thank you very much @grolimur !

File: README.md [hunk 2]
```diff
@@ -76,4 +76,45 @@ Alle  können hier direkt Fehler melden und dazu einfach einen [Issue](https://g
...
+</details>
+
+---
+
+<details>
+<summary>🇫🇷 Version française (cliquez ici)</summary>
+
+## Qu'est-ce que zotkat ?
+
+Zotkat `[:tzo:tkat]` est une extension de Zotero pour le catalogage au sens large et contient des **éléments** pour travailler les métadonnées. Cette extension peut par exemple servir au catalgage dans les centres de documentation spécialisés et bibliothèques ou à la conversion des métadonnées pour leur ajout dans l'index d'un outil de découverte.
```
Comment (stweil) [inline README.md:88]: s/catalgage/catalogage?

File: README.md [hunk 3]
```diff
@@ -76,4 +76,45 @@ Alle  können hier direkt Fehler melden und dazu einfach einen [Issue](https://g
...
+3. Copiez les fichiers javascript (pas la version HTML) des convertisseurs de zotkat
+4. Redémarrez Zotero
+5. Allez dans les `Préférences...` > `Exportation` > `Format par défaut` et choisissez le nouveau convertisseur à cet endroit-là (si la liste est vide ou si les nouvelles options n'apparaissent pas, fermez et rouvrez les préférences)
+
+## Quel convertisseur existe ?
+
+Actuellement, zotkat comprend les convertisseurs suivants:
+ * [`BIBFRAME.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/BIBFRAME.js) : Il s'agit d'un convertisseur dont le but est d'enregistrer les données dans le format BIBFRAME dans Zotero. Idéalement, ce convertisseur devrait être étendu pour offrir des fonctions d'import et d'export.
+ * [`MARC21XML.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/MARC21XML.js) : Il s'agit d'un convertisseur d'export dont le but est d'enregistrer les données de Zotero au format MARCXML. Idéalement, cela devrait être intégrer dans le convertisseur existant pour MARCXML (cf. [https://github.com/zotero/translators/issues/762](https://github.com/zotero/translators/issues/762)).
```
Comment (stweil) [inline README.md:103]: s/intégrer/intégré ?

File: README.md [hunk 4]
```diff
@@ -76,4 +76,45 @@ Alle  können hier direkt Fehler melden und dazu einfach einen [Issue](https://g
...
+ * [`PicaSWB.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/PicaSWB.js) : Il s'agit d'un convertisseur d'export dont le but est d'enregistrer les données de Zotero au format Pica tel qu'il est utilisé dans le réseau des bibliothèques du sud-ouest de l'Allemagne (Südwestdeutschen Bibliotheksverbundes - SWB) et être p. ex. importer dans le client WinIBW. Voir aussi l'article paru dans [LIBREAS #29](http://libreas.eu/ausgabe29/05kim/) (en allemand).
```
Comment (stweil) [inline README.md:104]: s/Südwestdeutschen/Südwestdeutscher

File: README.md [hunk 5]
```diff
@@ -76,4 +76,45 @@ Alle  können hier direkt Fehler melden und dazu einfach einen [Issue](https://g
...
+ * [`PicaGBV.js`](https://raw.githubusercontent.com/UB-Mannheim/zotkat/master/PicaGBV.js) : Il s'agit d'un convertisseur d'export dont le but est d'enregistrer les données de Zotero au format Pica tel qu'il est utilisé dans le catalgue commun des bibliothèques allemandes (Gemeinsamer Bibliotheksverbund - GBV). Ce convertisseur est adapté pour le cas particulier du catalogage les chapitres d'ebooks.
```
Comment (stweil) [inline README.md:105]: s/catalgue/catalogue

File: README.md [hunk 6]
```diff
@@ -76,4 +76,45 @@ Alle  können hier direkt Fehler melden und dazu einfach einen [Issue](https://g
...
+ 
+## Comment signaler une erreur ou la corriger ?
+
+Tout le monde peut signaler des erreurs ici en ouvrant une *[issue](https://github.com/UB-Mannheim/zotkat/issues)*. Les demandes de fonctionnalités peuvent également être discutées ici. Toute modification d'un convertsieeur peut être proposée en faisant une *pull reqest* directement sur GitHub. Une description plus détaillée peut être trouvée sur le [wiki](https://github.com/UB-Mannheim/zotkat/wiki) (en allemand).
+
+
+## Licences
+
+* Ces convertisseurs sont publiés sous licence [AGPL 3.0](https://github.com/UB-Mannheim/zotkat/blob/master/LICENSE) comme le sont tous les convertisseurs de Zotero.
+* Le script bashest simplement publié sous licence CC0.
```
Comment (stweil) [inline README.md:117]: s/bashest/bash est/

**Pull request merged**